src/applications/gi/components/vet-tec/VetTecProgramSearchResult.jsx

```diff
@@ -0,0 +1,101 @@
+import React from 'react';
+import { Link } from 'react-router';
+
+import { formatCurrency } from '../../utils/helpers';
+import {
+  renderCautionFlag,
+  renderSchoolClosingFlag,
+  renderPreferredProviderFlag,
+} from '../../utils/render';
+
+class VetTecProgramSearchResult extends React.Component {
+  render() {
+    const { version, result } = this.props;
+    const {
+      facilityCode,
+      description,
+      institutionName,
+      city,
+      state,
+      country,
+      tuitionAmount,
+      lengthInWeeks,
+      vaBah,
+      dodBah,
+    } = result;
+
+    const housing = Math.min(dodBah, vaBah);
+
+    const linkTo = {
+      pathname: `profile/${facilityCode}/${description}`,
+      query: version ? { version } : {},
+    };
+
+    return (
+      <div className="search-result">
+        <div className="outer">
+          {renderSchoolClosingFlag(this.props.result)}
+          {renderCautionFlag(this.props.result)}
+          <div className="inner">
+            <div className="row">
+              <div className="small-12 usa-width-seven-twelfths medium-7 columns">
+                <h2>
+                  <Link to={linkTo}>{description}</Link>
+                </h2>
+                <div style={{ position: 'relative', bottom: 0 }}>
+                  <p className="institution-name vads-u-font-weight--bold">
+                    {institutionName}
+                  </p>
+                  <p className="institution-location">{`${city}, ${state ||
+                    country}`}</p>
```

Comment:
SearchResult.jsx is using locationInfo from helpers.js to handle possible empty city/state/country values. Not pretty, but avoids leading or trailing commas.